Common/src/main/java/model/BookData.java

```diff
@@ -1,17 +1,42 @@
 package model;
 
-import lombok.AllArgsConstructor;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Builder;
 import lombok.ToString;
 
 /**
  * Class responsible to build the data of books that we need
```

Comment:
Class? Really? Surprised me here. Thought it was an :alien: mothership for a moment.